tests/orchestrator/workflows/core/test_events.py

```diff
@@ -105,6 +106,39 @@ def test_node_state_changes_as_a_result_of_standard_lifecycle_delete_shorthand_n
         ctx, interface_name=SHORTHAND_NAME, op_name='delete', executor=executor)
     _assert_node_state_changed_as_a_result_of_standard_lifecycle_operation(node, 'delete')
 
+def test_node_state_changes_as_a_result_of_standard_lifecycle_create_typequalified_name(ctx,
```

Comment:
Basically this tests module tests the events and the state changes associated with them. Since the shorthand_name and the type_qualified_name are handled the same in that regard, maybe it'd be better to write tests for `get_type_by_full_or_shorthand_or_typequalified_name` and `convert_shorthand_typequalified_to_full_type_name`, and place these under `tests/parser/test_tosca_simple_v1_0/presentation/types.py`, and move the `test_tosca_simple_v1_0.py` module to be under `tests/parser/test_tosca_simple_v1_0/end2end.py`